packages/next-auth/package.json

```
@@ -119,7 +119,7 @@
     "jest-environment-jsdom": "^28.1.1",
     "jest-watch-typeahead": "^1.1.0",
     "msw": "^0.42.3",
-    "next": "12.2.5",
+    "next": "^12.2.5",
```

```suggestion
    "next": "12.2.5",
```

This is a dev dependency and can stay as-is.